cli/src/alluxio.org/cli/cmd/fs/checkcaching.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package fs
+
+import (
+    "alluxio.org/cli/env"
+    "github.com/spf13/cobra"
+    "strconv"
+)
+
+func CheckCaching(className string) env.Command {
+    return &CheckCachingCommand{
+        BaseJavaCommand: &env.BaseJavaCommand{
+            CommandName:   "checkCaching",
```

Comment:
```suggestion
            CommandName:   "check-cached",
```

use `-` separator for CLI commands. also change the english verb tense to be more accurate